test_suite/generic/test_generic.py

```diff
@@ -310,6 +310,16 @@ def test_rpm_v_unsatisfied_dependencies(self, host):
             assert 'Unsatisfied' not in host.run('rpm -Va').stdout, \
                 'There are unsatisfied dependencies'
 
+    @pytest.mark.run_on(['all'])
+    def test_no_sshkeys_knownhosts(self, host):
+        '''
```

Comment:
Let's use double quotes here to be consistent.

```suggestion
        """
```